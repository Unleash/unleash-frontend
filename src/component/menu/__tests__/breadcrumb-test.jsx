import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Breadcrumb from '../breadcrumb';

test('breadcrumb for /features', () => {
    const tree = renderer.create(
        <MemoryRouter initialEntries={['/features']}>
            <Breadcrumb />
        </MemoryRouter>
    );

    expect(tree).toMatchSnapshot();
});

test('breadcrumb for /features/view/Demo', () => {
    const tree = renderer.create(
        <MemoryRouter initialEntries={['/features/view/Demo']}>
            <Breadcrumb />
        </MemoryRouter>
    );

    expect(tree).toMatchSnapshot();
});

test('breadcrumb for /strategies', () => {
    const tree = renderer.create(
        <MemoryRouter initialEntries={['/strategies']}>
            <Breadcrumb />
        </MemoryRouter>
    );

    expect(tree).toMatchSnapshot();
});
