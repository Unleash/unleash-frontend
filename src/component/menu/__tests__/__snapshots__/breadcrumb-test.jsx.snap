// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breadcrumb for /features 1`] = `
<span>
  <span
    className="headerTitleLink"
  >
    Feature Toggles
  </span>
</span>
`;

exports[`breadcrumb for /features/view/Demo 1`] = `
<span>
  <a
    className="headerTitleLink"
    href="/features"
    onClick={[Function]}
  >
    Feature Toggles
  </a>
  <span>
    <span>
       â€º 
    </span>
    <span
      className="headerTitleLink"
    >
      Demo
    </span>
  </span>
</span>
`;

exports[`breadcrumb for /strategies 1`] = `
<span>
  <span
    className="headerTitleLink"
  >
    Strategies
  </span>
</span>
`;
